src/explorer/models/workspaceNode.ts

```diff
@@ -33,19 +33,59 @@ export class WorkspaceNode extends NodeBase {
     const children = [];
     let node: RootNode;
 
-    node = new RootNode("Classes", "", "dataRootNode:classesRootNode", "CLS", this.options);
+    node = new RootNode(
+      "Classes",
+      "",
+      "dataRootNode:classesRootNode",
+      "CLS",
+      this.options,
+      false
+      //new vscode.ThemeIcon("symbol-class")
+    );
     children.push(node);
 
-    node = new RootNode("Routines", "", "dataRootNode:routinesRootNode", "RTN", this.options);
+    node = new RootNode(
+      "Routines",
+      "",
+      "dataRootNode:routinesRootNode",
+      "RTN",
+      this.options,
+      false
+      //new vscode.ThemeIcon("note")
```

Comment:
Why commented?

Reply:
Because at the moment there seems to be a tree bug which messes up alignment if some items (i.e. local files) have icons and others (remote) don't. I only see this bug when the containing folder has an icon.

I plan to report this on the VSCode project. Assuming it gets fixed we can uncomment the lines that add icons to our folders.